content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Planternetes"
+date: 2023-08-15T12:00:00+0000
+slug: kubernetes-v1-28-release
+---
+
+**Authors**: [Kubernetes v1.28 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.28/release-team.md)
+
+Announcing the release of Kubernetes v1.28 Planternetes, the second release of 2023!
+
+This release consists of 46 enhancements. 20 of those enhancements are entering Alpha, 14 have graduated to Beta, and 12 have graduated to Stable.
+
+## Release Theme And Logo
+
+**Kubernetes v1.28: _Planternetes_**
+
+The theme for Kubernetes v1.28 is *Planternetes*.
+
+{{< figure src="/images/blog/2023-08-15-kubernetes-1.28-blog/kubernetes-1.28.png" alt="Kubernetes 1.28 Planternetes logo" class="release-logo" >}}
+
+Each Kubernetes release is the culmination of the hard work of thousands of individuals from our community. The people behind this release are industry veterans, parents, others are students new to the ecosystem, contributing our unique experience to create a collective artifact with global impact.
```

Comment:
Could we make this sentence a little more comprehensive to include folks that don't fall into those three buckets?

Reply:
Thanks we have updated it.